diff -urN a/FreeFileSync/Source/ui/version_check_impl.h b/FreeFileSync/Source/ui/version_check_impl.h
--- a/FreeFileSync/Source/ui/version_check_impl.h	2017-02-08 12:45:14.000000000 +0100
+++ b/FreeFileSync/Source/ui/version_check_impl.h	2017-02-22 22:00:17.645718004 +0100
@@ -45,7 +45,6 @@
 }
 
 
-inline
 bool shouldRunPeriodicUpdateCheck(time_t lastUpdateCheck)
 {
     if (lastUpdateCheck == getVersionCheckInactiveId())
