diff -urN a/FreeFileSync/Source/ffs_paths.cpp b/FreeFileSync/Source/ffs_paths.cpp
--- a/FreeFileSync/Source/ffs_paths.cpp	2019-11-19 23:52:20.000000000 +0100
+++ b/FreeFileSync/Source/ffs_paths.cpp	2020-03-16 17:27:28.604644616 +0100
@@ -64,7 +64,11 @@
 
 Zstring fff::getResourceDirPf()
 {
-    return getProcessParentFolderPath() + FILE_NAME_SEPARATOR + Zstr("Resources") + FILE_NAME_SEPARATOR;
+    return getProcessParentFolderPath() + FILE_NAME_SEPARATOR
+        + Zstr("share") + FILE_NAME_SEPARATOR
+        + Zstr("FreeFileSync")+ FILE_NAME_SEPARATOR
+        + Zstr("Resources") + FILE_NAME_SEPARATOR
+    ;
 }
 
 
