diff -urN a/FreeFileSync/Source/base/ffs_paths.cpp b/FreeFileSync/Source/base/ffs_paths.cpp
--- a/FreeFileSync/Source/base/ffs_paths.cpp	2019-11-19 23:52:20.000000000 +0100
+++ b/FreeFileSync/Source/base/ffs_paths.cpp	2020-03-16 17:27:28.604644616 +0100
@@ -64,7 +64,11 @@
 
 Zstring fff::getResourceDirPf()
 {
-    return getProcessParentFolderPath() + FILE_NAME_SEPARATOR + Zstr("Resources") + FILE_NAME_SEPARATOR;
+    return getProcessParentFolderPath() + FILE_NAME_SEPARATOR
+        + Zstr("share") + FILE_NAME_SEPARATOR
+        + Zstr("FreeFileSync")+ FILE_NAME_SEPARATOR
+        + Zstr("Resources") + FILE_NAME_SEPARATOR
+    ;
 }
 
 
diff -urN a/zen/ring_buffer.h b/zen/ring_buffer.h
--- a/zen/ring_buffer.h	2019-11-19 23:52:19.000000000 +0100
+++ b/zen/ring_buffer.h	2020-03-16 17:27:59.921309571 +0100
@@ -8,6 +8,7 @@
 #define RING_BUFFER_H_01238467085684139453534
 
 #include <cassert>
+#include <cstddef>
 #include "scope_guard.h"
 
 
