diff -urN a/zen/build_info.h b/zen/build_info.h
--- a/zen/build_info.h	2019-10-17 18:37:03.000000000 +0200
+++ b/zen/build_info.h	2019-11-10 13:42:11.467093184 +0100
@@ -7,7 +7,7 @@
 #ifndef BUILD_INFO_H_5928539285603428657
 #define BUILD_INFO_H_5928539285603428657
 
-    #include <bit> //std::endian
+//    #include <bit> //std::endian
 
 #define ZEN_ARCH_32BIT 32
 #define ZEN_ARCH_64BIT 64
@@ -24,7 +24,8 @@
 
 constexpr bool usingLittleEndian()
 {
-    return std::endian::native == std::endian::little;
+    //return std::endian::native == std::endian::little;
+    return true;
 }
 
 #endif //BUILD_INFO_H_5928539285603428657
